static/home/css/style.css

```diff
@@ -101,17 +116,21 @@ nav{
 	top: 0;
 	background-color: #ff8c00;
 	z-index: 2;
+	font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;	
 }
+
 .navbut{
-	border: 1px red solid;
+	border: 1px black solid;
 	border-radius: 4px;
 	background-color: #ff8c00;
 	padding: 3px 10px;
 	cursor: pointer;
 	transition: 0.5s;
+	margin: 5.5px;
+	
 }
 .navbut:hover{
-	background-color: red;
+	background-color:red;
```

Comment:
Don't remove the spacing between both. It's how the code is formatted throughout the project.